physlr/physlr.py

```diff
@@ -1726,19 +1726,32 @@ def subgraph_stats_process(u):
         return Physlr.subgraph_stats(Physlr.graph, u)
 
     def physlr_subgraphs_stats(self):
-        "Retrieve subgraphs' stats."
-        gin = self.read_graph(self.args.FILES)
-        Physlr.filter_edges(gin, self.args.m)
-        print(
-            int(timeit.default_timer() - t0),
-            "Computing statistics of the subgraphs...", file=sys.stderr)
+        "Retrieve statistics of subgraphs."
+        nodes_of_interest = []
+        if len(self.args.FILES) > 1:
+            gin = self.read_graph([self.args.FILES[0]])
+            with open(self.args.FILES[1]) as fin:
+                for line in fin:
+                    nodes_of_interest.append(line.split()[0])
```

Comment:
Can you print one line of the file so I can know the format

Reply:
```suggestion
                    nodes_of_interest.append(line.strip().split()[0])
```
Is it possible for there to just be entry on the line? I guess goes along with Johnathan's question

Reply:
@jowong4 @lcoombe Can be in any format by with the barcode in the first column, like:
`ACGTACGTACGT-1  100`
or just the entry alone:
`ACGTACGTACGT-1`

@lcoombe What does a call to `strip()` with no arguments do?

Reply:
@afshinfard - It just strips the white space from the beginning and end of the string (like newlines at the end of a line)